docs/ldml/tr35.md

```diff
@@ -1350,38 +1350,37 @@ Hybrid locales have intermixed content from 2 (or more) languages, often with on
 
-To communicate requests for localized content and internationalization services, locales are used. When people pick a language from a menu, internally they are picking a locale (en-GB, es-419, etc.). To allow an application to support Spanglish or Hinglish locale selection, [unicode_locale_id](#unicode_locale_id)s can represent hybrid locales using the T extension key-value 'h0-hybrid'. (For more information on the T extension, see _Section 3.7 [Unicode BCP 47 T Extension](#t_Extension)._)
+To communicate requests for localized content and internationalization services, locales are used. When people pick a language from a menu, internally they are picking a locale (en-GB, es-419, etc.). To allow an application to support Spanglish or Hinglish locale selection, [unicode_locale_id](#unicode_locale_id)s can represent hybrid locales using the T Extension key-value 'h0-hybrid'. (For more information on the T extension, see _Section 3.7 [Unicode BCP 47 T Extension](#t_Extension)._)
 
 _However, if users typically expect a their language in non-default script to contain a significant amount of text due to lexical borrowing, then the -t- and hybrid subtags may be omitted. An example of this is when Hindi is written in Latin script since Romanized Hindi typically contains a significant amount of English text, ‘hi-Latn’ can be used instead of ‘hi-Latn-t-en-h0-hybrid’._
 This tends to work better in implementations that don't yet handle the -t- extension.
 
 Examples:
 
-<!-- HTML: no header, colspan -->
-<table><tbody>
-<tr><td>hi-t-<u>en-h0-hybrid</u></td>   <td>Hinglish</td>   <td>Hindi-English hybrid locale</td></tr>
-<tr><td>ta-t-<u>en-h0-hybrid</u></td>   <td>Tanglish</td>   <td>Tamil-English hybrid locale</td></tr>
-<tr><td>ba-t-<u>en-h0-hybrid</u></td>   <td>Banglish</td>   <td>Bangla-English hybrid locale</td></tr>
-<tr><td colspan="3">…</td></tr>
-<tr><td>en-t-<u>hi-h0-hybrid</u></td>   <td>Hinglish</td>   <td>English-Hindi hybrid locale</td></tr>
-<tr><td>en-t-<u>zh-h0-hybrid</u></td>   <td>Chinglish</td>  <td>English-Chinese hybrid locale</td></tr>
-<tr><td colspan="3">…</td></tr>
-</tbody></table>
+|Hybrid locale code		| Hybrid name	| Description						|
+|-------------------------------|---------------|-------------------------------------------------------|
+|hi-t-***en-h0-hybrid***	| Hinglish	| Hindi-English hybrid locale written in Devanagari	|
+|hi-Latn-t-***en-h0-hybrid***	| Hinglish	| Hindi-English hybrid locale written in Latin script	|
+|hi-Latn			| Hinglish	| Hindi-English hybrid locale written in Latin script	|
+|ta-t-***en-h0-hybrid***  	| Tanglish	| Tamil-English hybrid locale written in Tamil script	|
+|...													||
+|en-t-***hi-h0-hybrid***	| Hinglish	| English-Hindi hybrid locale written in Latin script	|
```

Comment:
Suggestion: Clarify when you should prefer en-t-hi-h0-hybrid versus hi-t-en-h0-hybrid

Reply:
That's already in the note under the examples table, except that we are also fixing the ill-formed language tag syntax in that note.